src/components/FormInput.tsx

```diff
@@ -272,6 +280,18 @@ const FormInput = (props: InputProps) => {
   const { min = 1, max = 1 } = resultCount ?? {};
   const chipDataList: ChipData[] = getInitialChipDataList();
 
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => {
+    setIsFocused(true);
+    setIsInputFocused?.(true);
+  };
+
+  const handleBlur = () => {
+    setIsFocused(false);
+    setIsInputFocused?.(false);
+  };
```

Comment:
```suggestion
  const handleFocus = () => {
    setIsFocused(true);
    onBlur?.(true);
  };

  const handleBlur = () => {
    setIsFocused(false);
    onBlur?.(false);
  };
```

setIsInputFocused: 이건 input 에 focused 된거
setIsFocused : ?

둘다 focused 상태 관리하는것 같은데 둘 차이가 좀 모호해서, 그냥 onBlur 를 받는게 더 나아보입니당

Reply:
onBlur 가 focus 가 아닌 상태를 의미하는데 focus 일 때 onBlur?.(true) 를 주는게 안 맞는것 같습니다.
제가 생각한 setIsInputFocused 는 폼에서 어떤 인풋이 focus 가 된 상태를 생각해서 잘 맞는 이름이라 생각했어요

Reply:
좀더 구분짓자면 저는 setIsAnyInputFocused 라고 해도 괜찮아보입니다

Reply:
아 onBlur 가 아닌 onFocused 가 맞네요 comment 수정했습니다 머릿속에 blur 가 박혔었나봅니다 😅

둘이 그 안에서 굳이 구분지으려하는 것 보단 외부에서 이 FormInput component 를 사용할땐 focused 상태엔 뭐든 받아서 넘길 수 있도록 좀더 제너럴 하게 만드는게 낫지 않을까 했던 취지였습니다.